src/hdmf/term_set.py

```diff
@@ -94,3 +115,39 @@ def __getitem__(self, term):
         except KeyError:
             msg = 'Term not in schema'
             raise ValueError(msg)
+
+    def schemasheets_convert(self):
```

Comment:
Please add a docstring for `` schemasheets_convert`` to indicate what the function does